### PR TITLE
BIGTOP-3711: Add read/write smoke tests for Kafka

### PR DESCRIPTION
**Pull request by yoda-mon**
### Description of PR

This PR adds read/write tests for an Kafka topic to improve the test coverage.
- Add a write test using `kafka-console-producer.sh`.
- Add a read test using `kafka-console-consumer.sh`. This test try to read the message written by the write test above.

### How was this patch tested?

```sh
./docker-hadoop.sh -d   -C config_ubuntu-20.04.yaml   --create 3   --memory 8g   --disable-gpg-check   --stack zookeeper,kafka   --smoke-tests kafka
``` 
### For code changes:

- [x] Does the title or this PR starts with the corresponding JIRA issue id (e.g. 'BIGTOP-3638. Your PR title ...')?
- [x] Make sure that newly added files do not have any licensing issues. When in doubt refer to https://www.apache.org/licenses/

### CODE REVIEW

**Comment by yoda-mon** [comment]
Sorry for confusing, when you try to test this PR with locally build zookeeper, it will be failed by [BIGTOP-3712](https://issues.apache.org/jira/browse/BIGTOP-3712). Please review the [PR](https://github.com/apache/bigtop/pull/925) for the issue before this.

File: bigtop-tests/smoke-tests/kafka/TestKafkaSmoke.groovy [hunk 1]
```diff
@@ -44,9 +51,27 @@ class TestKafkaSmoke {
...
+    sh.exec("echo '" + TEST_MESSAGE + "' | " + KAFKA_PRODUCER + " --broker-list localhost:9092 --topic test");
+    assertTrue(
+      " Write Kafka topics failed. " + sh.getOut() + " " + sh.getErr(),
+      sh.getRet() == 0
+    );
+  }
+
+  @Test
+  public void test2ReadKafkaTopics() {
+    sh.exec(KAFKA_CONSUMER + " --bootstrap-server localhost:9092 --topic test --partition 0 --offset 0 --max-messages 1");
```
Comment (iwasakims) [inline bigtop-tests/smoke-tests/kafka/TestKafkaSmoke.groovy:71]: `--timeout-ms` should be given to bail out fast in failure scenario?

Comment (yoda-mon) [reply bigtop-tests/smoke-tests/kafka/TestKafkaSmoke.groovy:71]: Thank you for your comment, I agree. I will try to add the option.

Comment (yoda-mon) [reply bigtop-tests/smoke-tests/kafka/TestKafkaSmoke.groovy]: I added `--timeout-ms 10000` to the consumer test and it passed the test.
On my environment, 100 ms failed and 1000 ms passed the test, so I set 10 times lager value.

**Comment by sekikn** [comment]
+1, tested on Fedora 35, Debian 10 and Ubuntu 18.04.

```
$ ./docker-hadoop.sh -d -c 3 -F docker-compose-cgroupv2.yml -i bigtop/puppet:trunk-fedora-35 -r file:///bigtop-home/output -G -k zookeeper,kafka -s kafka

...

> Task :bigtop-tests:smoke-tests:kafka:test
Finished generating test XML results (0.024 secs) into: /bigtop-home/bigtop-tests/smoke-tests/kafka/build/test-results/test
Generating HTML test report...
Finished generating test html results (0.043 secs) into: /bigtop-home/bigtop-tests/smoke-tests/kafka/build/reports/tests/test
Now testing...
:bigtop-tests:smoke-tests:kafka:test (Thread[Daemon worker,5,main]) completed. Took 10.543 secs.

BUILD SUCCESSFUL in 1m 47s
34 actionable tasks: 7 executed, 27 up-to-date
```

```
$ ./docker-hadoop.sh -d -c 3 -i bigtop/puppet:trunk-debian-10 -r file:///bigtop-home/output/apt -G -k zookeeper,kafka -s kafka

...

> Task :bigtop-tests:smoke-tests:kafka:test
Finished generating test XML results (0.017 secs) into: /bigtop-home/bigtop-tests/smoke-tests/kafka/build/test-results/test
Generating HTML test report...
Finished generating test html results (0.032 secs) into: /bigtop-home/bigtop-tests/smoke-tests/kafka/build/reports/tests/test
Now testing...
:bigtop-tests:smoke-tests:kafka:test (Thread[Daemon worker,5,main]) completed. Took 10.018 secs.

BUILD SUCCESSFUL in 1m 9s
34 actionable tasks: 7 executed, 27 up-to-date
```

```
$ ./docker-hadoop.sh -d -c 3 -i bigtop/puppet:trunk-ubuntu-18.04 -r file:///bigtop-home/output/apt -G -k zookeeper,kafka -s kafka

...

> Task :bigtop-tests:smoke-tests:kafka:test
Finished generating test XML results (0.018 secs) into: /bigtop-home/bigtop-tests/smoke-tests/kafka/build/test-results/test
Generating HTML test report...
Finished generating test html results (0.045 secs) into: /bigtop-home/bigtop-tests/smoke-tests/kafka/build/reports/tests/test
Now testing...
:bigtop-tests:smoke-tests:kafka:test (Thread[Execution worker for ':',5,main]) completed. Took 11.816 secs.

BUILD SUCCESSFUL in 1m 8s
34 actionable tasks: 7 executed, 27 up-to-date
```

**Comment by sekikn** [comment]
Merged into master. Thanks @yoda-mon!

**Pull request merged**